module/input/NatNet/src/Parse.hpp

```diff
@@ -154,9 +154,11 @@ namespace module::input {
             rigidBody.id       = ReadData<uint32_t>::read(ptr, version);
             rigidBody.position = ReadData<Eigen::Matrix<float, 3, 1>>::read(ptr, version);
             rigidBody.rotation = ReadData<Eigen::Matrix<float, 4, 1>>::read(ptr, version);
-            rigidBody.markers  = ReadData<std::vector<Marker>>::read(ptr, version);
 
             // Version specific information
+            if (version < 0x03000000) {
+                rigidBody.markers = ReadData<std::vector<Marker>>::read(ptr, version);
```

Comment:
how are rigid body markers read now?

Reply:
According to the documentation the marker positions have been removed as redundant as they can be derived from the position and orientation plus offset to optimize packet size